pacta_data_qa.Rmd

```diff
@@ -0,0 +1,660 @@
+---
+title: "PACTA Analysis - Input Data QA"
+date: "`r format(Sys.time(), '%B %Y')`"
+params: 
+  project_code: general
+  project_location_ext: NULL
+  twodii_internal: TRUE
+  project_name: NULL
+output:
+  html_document:
+    fig_caption: yes
+    theme: flatly
+    toc: yes
+    toc_depth: 3
+    toc_float: yes
+---
+
+```{r setup, include=FALSE}
+# set echo = TRUE, for more insight to the code printed in the output
+knitr::opts_chunk$set(echo = FALSE)
+library(tidyverse)
+library(DT)
+
+project_code <- params$project_code
+project_location_ext <- params$project_location_ext
+twodii_internal <- params$twodii_internal
+project_name <- params$project_name
+
+create_project_folder(project_name, twodii_internal, project_location_ext)
+set_project_paths(project_name, twodii_internal, project_location_ext)
```

Comment:
will these functions be available when this runs? seems like we would need to source the R scripts or run `devtools::load_all()` or something to make these available